src/app/_components/Feedback/Feedback.js

```diff
@@ -0,0 +1,146 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import LoveIcon from '@icons/loveIcon.svg';
+import { useCreateFeedback } from '@/app/_hooks';
+import SendFeedback from '@/lib/validationSchemas/sendFeedbackSchema';
+import { Modal } from '../Modal';
+import { TextInputField } from '../InputFields';
+import { CheckBox } from '../CheckBox';
+import { TextArea } from '../TextArea';
+import { PillButton } from '../PillButton';
+import { cn } from '@/utils/cn';
+import { buttonColorVariant, buttonVariant } from '../PillButton/style';
+
+const MAX_LENGTH = 320;
+
+export function Feedback({ isFeedbackOpen, onClose }) {
+  const [isFormOpen, setFormOpen] = useState(true);
+
+  const [name, setName] = useState('');
+  const [phone, setPhone] = useState('');
+  const [email, setEmail] = useState('');
+  const [isChecked, setChecked] = useState(true);
+  const [message, setMessage] = useState('');
+
+  const [validationErrors, setValidationErrors] = useState({});
+
+  const { mutate: createFeedback } = useCreateFeedback();
+
+  const onSubmit = e => {
+    e.preventDefault();
+    const formData = {
+      name,
+      phone,
+      callMe: isChecked,
+      email: email.length ? email : undefined,
```

Comment:
🟢 would be nice to have little comment about why is that needed

Reply:
Just an alternative option:
```js
email: email?.length // either email.length or undefined
```

Reply:
**_added an explanatory comment to the code:_**  
```
 // I am forced to do exactly that, since I have to pass a valid email or "undefined" to the backend.
 // Input should have base value ""
```